.github/workflows/lint.yml

```diff
@@ -1,9 +1,19 @@
 name: Lint
-on: [push]
+on:
+  push:
 env: 
   CARGO_TERM_COLOR: always
 
 jobs:
+  actionlint: 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+      - name: Run actionlint
+        uses: devops-actions/actionlint@v0.1.3
+        with:
+          shellcheck_opts: '-e SC2086'
```

Comment:
A bunch of violations of this https://www.shellcheck.net/wiki/SC2086 which I'm going to postpone to a future PR